04_Running_The_Class.Rmd

```diff
@@ -0,0 +1,285 @@
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+## Add Student Group to Workspace(s)
+
+Now that you have added students to the Group, you can easily share Terra resources with the entire Group.
+
+First, you will need to give your Student Group access to any Workspaces that they will need.
+
+If you are **using a Public Workspace** (most pre-made content uses Public Workspaces), **you can skip this step** - Public Workspaces are automatically available to everyone on AnVIL.  
+
+If you have **Workspaces of your own** that your students will need to access, you should **add the Student Group to your Workspace as a *Reader***.
+
+- Adding students as a Reader gives them permission to view the Workspace, and (once they have funding) to **clone their own copy**.  This will enable them to work safely in their own space without interfering with each other or overwriting anything in the original Workspace.
+- Adding students to your Workspace(s) can safely be done ahead of time; as Readers they can't run computations and run up a bill.
+
+To add the Student Group to the Workspace, you will need the Group email address, which can be found on the Group management page: https://anvil.terra.bio/#groups
+
+The following instructions explain how to add someone to a Terra Workspace.  You will want to add students to the Workspace as **Readers**.
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_workspace_share.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+Make sure the newly added Student Group displays "Reader" under "Roles".
+
+# Teach the Class
+
+This chapter covers things that you will generally do on the day of the class:
+
+<br>
+
+**Before class**
+
+- Add students to Billing Project
+- Direct students to clone the Workspace
+
+We generally recommend waiting until shortly before the class to add students to the Billing Project, to prevent them from starting something up early and accruing costs.  *However, if you did not create the Terra Billing Project for your class, you should make sure that either you have been given "Owner" permission for the class Billing Project (so you can add students yourself), or that someone who does is prepared to add students in time for your class to start.*
+
+We typically have students clone the Workspace in the first few minutes of class.  This both helps ensure they won't start up their cloud environments early, and ensures they get the most up-to-date version of the Workspace.
+
+**After class**
+
+- Collect feedback
+- Give students any necessary instructions on how to save and export any important files, and how to pause or delete their cloud environments
+
+## Before: Add Students to Billing Project
```

Comment:
I think we should be opinionated and encourage instructors to use the student group wherever possible. This section could refer to the Student Group and not individual students.